neard/src/runtime/errors.rs

```diff
@@ -0,0 +1,84 @@
+use near_chain::near_chain_primitives::error::QueryError;
+
+#[derive(thiserror::Error, Debug)]
+#[error(transparent)]
+pub struct RuntimeQueryError(#[from] pub QueryError);
```

Comment:
Do we need to be able to construct `RuntimeQueryError` from the `QueryError`?

```suggestion
pub struct RuntimeQueryError(pub QueryError);
```

I also suggest we rename it to `WrappedQueryError` to better indicate the intention that we just want to have a wrapped type here, and also add a comment explaining that we cannot implement `From<>` trait for the original `QueryError` struct since it is a foreign type.